src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/main.js

```diff
@@ -127,6 +127,19 @@ switch (testCase) {
         };
         dotnet.withConfig({ maxParallelDownloads: maxParallelDownloads });
         break;
+    case "AllocateLargeHeapThenInterop":
+        dotnet.withEnvironmentVariable("MONO_LOG_LEVEL", "debug")
+        dotnet.withEnvironmentVariable("MONO_LOG_MASK", "gc")
+        dotnet.withModuleConfig({
+            preRun: (Module) => {
+                // wasting 2GB of memory
+                for (let i = 0; i < 210; i++) {
+                    testOutput(`wasting 1m ${Module._malloc(10 * 1024 * 1024)}`);
```

Comment:
nit

```suggestion
                    testOutput(`wasting 10m ${Module._malloc(10 * 1024 * 1024)}`);
```

Reply:
What's `m`?

Reply:
My guess is millions, like mega, but 🤷‍♂️